### ISSUE TITLE
mediator module is missing

### ISSUE THREAD

**Issue by nikosT** [EIDA/ws-tests#1]
When trying to run, I get the error:
`ImportError: No module named mediator
`

**Comment by jan-michalek** [comment]
Yes, I experienced the same issue. Any suggestions to get through? Do I need to install mediator (I don't think it should be required)?
